test/basic.html

```diff
@@ -101,6 +109,13 @@
         assert.notEqual(getComputedStyle(error).visibility, 'hidden', 'error is not visibility:hidden');
         assert.isTrue(container.invalid);
       });
+
+      test('input format respects parentheses and spaces', function() {
```

Comment:
Please add tests for incomplete matching too. Like, if I start with `11111`, does it get formatted to `(111) 11`